flexget/api/core/database.py

```diff
@@ -44,19 +46,19 @@ def post(self, session=None):
             plugin_name = data.get('plugin_name')
             if not plugin_name:
                 raise BadRequest(
-                    '\'plugin_name\' attribute must be used when trying to reset plugin'
+                    "'plugin_name' attribute must be used when trying to reset plugin"
                 )
             try:
                 reset_schema(plugin_name)
-                msg = 'Plugin {} DB reset was successful'.format(plugin_name)
+                msg = f'Plugin {plugin_name} DB reset was successful'
             except ValueError:
-                raise BadRequest('The plugin {} has no stored schema to reset'.format(plugin_name))
+                raise BadRequest(f'The plugin {plugin_name} has no stored schema to reset')
         return success_response(msg)
 
 
 @db_api.route('/plugins/')
 class DBCleanup(APIResource):
     @api.response(200, model=plugins_schema)
-    def get(self, session=None):
+    def get(self, session: None = None) -> Response:
```

Comment:
type shouldn't be None, it's Session

Reply:
`session` gets injected by the `@with_session` decorator, which is not applied to this method.

Reply:
`with_session` is implicitly called for all `APIResource` method via the `method_decorators` attribute: https://github.com/Flexget/Flexget/blob/5e3b8156247887df8fa74fd26205bb6c2055c186/flexget/api/app.py#L84

Reply:
You are correct (just verified by looking into the flask_restx `Resource` implementation). All such annotations have been changed to use `sqlalchemy.orm.Session`, adding the import wherever required.